CharacterCreation/00_New_Player_Walkthrough.txt

```diff
@@ -31,17 +31,17 @@ on what the attributes themselves do and the dependencies of the skills you sele
 between 7-9 each. Then give a 3-5 to one or two things your character won't be as good at, and finally adjust your numbers 
 as necessary to give medium scores to the remaining attributes.
 
-##Step 5: Feats
+## Step 5: Feats
 If you are making a level 1 character, you do not need to worry about feats. If your GM wants you to begin at a higher 
 level, you may wish to ask them for advice on which feats to take.
 
-##Step 6: Weapons, Attachments, and Armor
+## Step 6: Weapons, Attachments, and Armor
 Weapon and Armor proficiencies are determined by your class, as is your financial reach. Usually, though not always, you
 will want to acquire the heaviest type of armor your class has access to. Weapons are trickier, as each type of weapon
 is the best class of weapon for a particular task. As a [Class recommended in step 2], you may want a [X weapon for Y task],
 or a [Z weapon for W task]. [Sidearm discussion (if money permits)].
 
-##Step 7: Equipment
+## Step 7: Equipment
 Any remaining moneys you have can be spent on miscellaneous equipment from the [Items](https://github.com/bleehu/Compound_X/blob/play/CharacterCreation/07_Items.txt) document. Take whatever items you
```

Comment:
This should probably say 'credits' instead of money, but that's not a big change.

Reply:
Good Aesthetic Catch!